src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Logs/FilterLogsByAttributeType.php

```diff
@@ -77,6 +77,6 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getBlockPrefix()
     {
-        return 'logs_filters_block';
+        return 'logs';
```

Comment:
we have a sort of conventions here: the form used *must* have the same name than the grid name.

It's a limitation I want to remove later, not really a blocking issue for the migration :)

Reply:
limitation removed :)